pandera/backends/polars/container.py

```diff
@@ -0,0 +1,113 @@
+"""Validation backend for polars DataFrameSchema."""
+
+from typing import Optional, List
+
+import polars as pl
+
+from pandera.api.polars.container import DataFrameSchema
+from pandera.backends.base import BaseSchemaBackend, CoreCheckResult
```

Comment:
```suggestion
from pandera.backends.base import CoreCheckResult
```
Think it's unused; ran across it while referencing for Ibis backend.